### PR TITLE
Add "GuardDuty Detector Disabled" query for CloudFormation #2297

### LINKED ISSUE #2297: Add "GuardDuty Detector Disabled" query for CloudFormation
### Platform
CloudFormation

### Provider
Aws

### Description
Make sure that Amazon GuardDuty is Enabled.

### PR DESCRIPTION
**Pull request by pedro-mimoso**
Closes #2297 

Make sure that Amazon GuardDuty is Enabled.

### CODE REVIEW

File: assets/queries/cloudFormation/guardduty_detector_disabled/query.rego [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+	resource.Type == "AWS::GuardDuty::Detector"
+	properties := resource.Properties
+	properties.Enable == false
+
+	result := {
+		"documentId": input.document[i].id,
+		"searchKey": sprintf("Resources.%s.Properties.Enable", [name]),
+		"issueType": "IncorrectValue",
+		"keyExpectedValue": "GuardDuty Detector should be Enabled",
+		"keyActualValue": "GuardDuty Detector is not Enabled",
```
Comment (Ruben-Silva) [inline assets/queries/cloudFormation/guardduty_detector_disabled/query.rego:14]: Change to:

"keyExpectedValue": sprintf("Resources.%s.Properties.Enable" is set to true, [name]),
		"keyActualValue": sprintf("Resources.%s.Properties.Enable" is set to false, [name]),

File: assets/queries/cloudFormation/guardduty_detector_disabled/test/negative2.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
+{
+  "document": [
+    {
+      "AWSTemplateFormatVersion": "2010-09-09",
+      "Resources": {
+        "mydetector": {
```
Comment (Ruben-Silva) [inline assets/queries/cloudFormation/guardduty_detector_disabled/test/negative2.json:6]: Change "mydetector" to "mydetector2"

File: assets/queries/cloudFormation/guardduty_detector_disabled/test/positive1.yaml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Resources:
+  mydetector:
```
Comment (Ruben-Silva) [inline assets/queries/cloudFormation/guardduty_detector_disabled/test/positive1.yaml:3]: Change "mydetector" to "mydetector3"

File: assets/queries/cloudFormation/guardduty_detector_disabled/test/positive2.json [hunk 4]
```diff
@@ -0,0 +1,12 @@
+{
+  "AWSTemplateFormatVersion": "2010-09-09",
+  "Resources": {
+    "mydetector": {
```
Comment (Ruben-Silva) [inline assets/queries/cloudFormation/guardduty_detector_disabled/test/positive2.json:4]: Change "mydetector" to "mydetector4"

**Review by Ruben-Silva — CHANGES_REQUESTED** [review]

**Review by Ruben-Silva — APPROVED** [review]
Looks good to me!

**Review by ruigomescx — APPROVED** [review]

**Pull request merged**